tools/shoestring/shoestring/wizard/setup_file_generator.py

```diff
@@ -115,3 +120,10 @@ async def prepare_shoestring_files(screens, directory):
 			replacements.append(('imports', 'harvester', str(config_harvesting_filepath)))
 
 	ConfigurationManager(config_filepath.parent).patch(config_filepath.name, replacements)
+
+
+async def patch_shoestring_config(shoestring_filepath, new_config_filepath):
```

Comment:
can you remind me what is "new" in this context? from package or from local machine?

Reply:
"new" is the downloaded package config...will update